pallets/remote-asset-manager/Cargo.toml

```diff
@@ -109,6 +109,10 @@ runtime-benchmarks = [
     'frame-benchmarking',
     'frame-support/runtime-benchmarks',
     'frame-system/runtime-benchmarks',
+    'pallet-collective/runtime-benchmarks',
+    # polkadot
+    'xcm-builder/runtime-benchmarks',
+    'pallet-xcm/runtime-benchmarks'
```

Comment:
this probably won't compile because these are `dev` deps only.

Reply:
ur right